guides/using-iron-icons.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Using Icons
+summary: "How to display and style icons, and how to use the Polymer team's icon collection in your own project."
+tags: ['icons','intermediate']
+elements: ['iron-icon','iron-icons', 'iron-iconset', 'iron-iconset-svg']
+updated: 2015-07-17
+---
+
+[//]: # (align docs with videos)
+[//]: # (figure out how to display iron-icons, provide a gist?)
+[//]: # (delete core-iconset video after aligning with docs)
+[//]: # (are ::shadow and /deep/ still supported?)
+[//]: # (load dependenices in right order https://github.com/PolymerElements/iron-icon/issues/19)
+[//]: # (get SVG icon set example working)
+[//]: # (use for displaying icons https://elements.polymer-project.org/elements/iron-icons?view=demo:demo/index.html)
+[//]: # (during review, ask about using multiple icon sets, specifying each with namespace syntax)
+[//]: # (instead of trying to display inside of document, create working examples with full index.html)
+[//]: # (how to style icons embedded in other elements?)
+[//]: # (delete video if it mentions iron-iconset*)
+[//]: # (color not inheriting on paper-icon-button)
+
+
+<link rel="import" href="/bower_components/google-youtube/google-youtube.html">
+
+## Introduction
+
+In this guide we will teach you how to display icons and how to use the 
+Polymer team's collection of icons in your project.
+
+We discuss two elements:
```

Comment:
`iron-icons` isn't an element (see discussion somewhere below). Maybe just: This guide discusses: